x-pack/test/api_integration/apis/cloud_security_posture/helper.ts

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { SuperTest, Test } from 'supertest';
+import { Client } from '@elastic/elasticsearch';
+import { SecurityService } from '../../../../../test/common/services/security/security';
+
+export const deleteIndex = (es: Client, indexToBeDeleted: string[]) => {
+  Promise.all([
+    ...indexToBeDeleted.map((indexes) =>
+      es.deleteByQuery({
+        index: indexes,
+        query: {
+          match_all: {},
+        },
+        ignore_unavailable: true,
+        refresh: true,
+      })
+    ),
+  ]);
+};
+
+export const addIndex = async <T>(es: Client, findingsMock: T[], indexName: string) => {
+  await Promise.all([
+    ...findingsMock.map((finding) =>
+      es.index({
+        index: indexName,
+        body: {
+          ...finding,
+          '@timestamp': new Date().toISOString(),
+        },
+        refresh: true,
+      })
+    ),
+  ]);
+};
+
+export async function createPackagePolicy(
+  supertest: SuperTest<Test>,
+  agentPolicyId: string,
+  policyTemplate: string,
+  input: string,
+  deployment: string,
+  posture: string
+) {
+  const version = posture === 'kspm' || posture === 'cspm' ? '1.2.8' : '1.3.0-preview2';
```

Comment:
We can stop using the preview of `1.3.0`.
```suggestion
  const version = '1.3.0';
```